helm-charts/common/nginx/README.md

```diff
@@ -2,6 +2,44 @@
 
 Helm chart for deploying OPEA nginx service.
 
-## Using nginx chart
+## Integration with OPEA Applications
 
-E2E charts for OPEA applications should provide ConfigMap of name `{{ .Release.Name }}-nginx-config` with the required environment variables to configure [OPEA nginx microservice](https://github.com/opea-project/GenAIComps/blob/main/comps/nginx/nginx.conf.template).
+When used as a simple proxy within OPEA application charts (ChatQnA, CodeGen, etc.), parent charts should provide a ConfigMap named `{{ .Release.Name }}-nginx-config` containing environment variables for the [OPEA nginx microservice](https://github.com/opea-project/GenAIComps/blob/main/comps/nginx/nginx.conf.template).
+
+## Gateway Mode (Central Router)
+
+The nginx chart can be configured to act as a central gateway/router for OPEA services. This mode provides:
+
+- Service routing to multiple OPEA microservices
+- Custom nginx configuration template support
+- Health check endpoints
+- Optional ingress and monitoring support
+
+### Quick Start
+
+```bash
+# Deploy as gateway
+helm install nginx-gateway . -f gateway-values.yaml
+
+# Access OPEA services through the gateway
+curl http://nginx-gateway/v1/chatqna -d '{"query": "Hello"}'
```

Comment:
this is the mega service endpoint. possible to also give a micro service endpoint example?